include/usb/class/usb_hid.h

```diff
@@ -44,11 +44,7 @@ typedef void (*hid_idle_cb_t)(const struct device *dev, uint16_t report_id);
  */
 struct hid_ops {
 	hid_cb_t get_report;
```

Comment:
I think Get/SetReport callbacks should also be removed because of the same reason and the callback signature for set_report is wrong as no data is transferred to host. If at some point an interface for feature reports is needed we can then implement it reasonably.